README.md

```diff
@@ -1,15 +1,15 @@
 # pg_failover_slots
 
-An extension that makes logical replication slots practically usable across
-physical failover.
+PG Failover Slots is for anyone with Logical Replication publications on Postgres databases that are also part of a Streaming Replication architecture.
 
-This extension does the following:
+PG Failover Slots (pg_failover_slots) is an extension released as open source software under the PostgreSQL LICENSE. If you have logical replication publications on Postgres databases that are also part of a streaming replication architecture, PG Failover Slots avoids the need for you to reseed your logical replication tables when a new standby gets promoted to primary.
```

Comment:
I don't like these changes because they change the language from talking about logical replication slots to logical replication publications. That's wrong, though because logical replication publications have nothing to do with slots.

Publications are a filtering mechanism that is used by the logical replication.

Slots are consumer tracking mechanism on upstream, they work both for logical replication and any other logical change consumers, say wal2json, and pg_failover_slots covers those slots too.